nano/node/bootstrap/bootstrap_bulk_pull.hpp

```diff
@@ -49,15 +49,16 @@ class bulk_pull_client final : public std::enable_shared_from_this<nano::bulk_pu
 	uint64_t unexpected_count;
 	bool network_error{ false };
 };
+class bootstrap_attempt_wallet;
 class bulk_pull_account_client final : public std::enable_shared_from_this<nano::bulk_pull_account_client>
 {
 public:
-	bulk_pull_account_client (std::shared_ptr<nano::bootstrap_client> const &, std::shared_ptr<nano::bootstrap_attempt> const &, nano::account const &);
```

Comment:
I may be misunderstanding this but it seems strange to me to limit the bulk pull to a wallet bootstrap attempt only.

Reply:
This is a bulk_pull_account call which is a different network messages from bulk_pull. It is only used by bootstrap_attempt_wallet.